Libplanet.Extensions.Cocona/Commands/KeyCommand.cs

```diff
@@ -323,16 +343,29 @@ private PrivateKey ValidateRawHex(string rawKeyHex)
             return key;
         }
 
-        private Stream PathHandler(string pathString)
+        private string ValidateJsonPath(string jsonPath)
+        {
+            try
+            {
+                string json = new StreamReader(jsonPath).ReadToEnd();
+                return json;
+            }
+            catch (Exception)
+            {
+                Utils.Error("This file does not exist.");
+                return string.Empty;
+            }
```

Comment:
If there is no file, it throws `System.IO.FIleNotFoundException`. And `Utils.Error` return `CommandExitedException`
```suggestion
            catch (System.IO.FileNotFoundException)
            {
                throw Utils.Error("This file does not exist.");
            }
```